Client/src/Components/Charts/Gauge/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { RadialBarChart, RadialBar, ResponsiveContainer } from "recharts";
+import PropTypes from "prop-types";
+
+const DATA = [{ value: 100 }];
+const PROGRESS_THRESHOLD = 50;
+const DEFAULT_CONTAINER_HEIGHT = 160;
+const DEFAULT_HEADER_FONT = { fontSize: "11px" };
```

Comment:
Is it possible to get this value from our theme?

Reply:
These are values specific to the guage component , I extract them out for readability. I dont think it makes sense to put them into theme, e.g, the default header font, I will rename it into default header, is only meant to provide a default font size value since I left gaugeHeader as optional, on purpose, user can provide what ever value that looks good on their setup , same for the DEFAULT_CONTAINER_HEIGHT  @ajhollid what do you think?

Reply:
Yes, but size and color values should live within the theme, so all application is pointing to the same place. If that comes a time where we will change some font size or color, it applies to all elements

Reply:
Hey @jennifer-gan, my thoughts are anything related to color and font should be in the theme, a default font size is a good idea, but it should be declared in the theme as well.

I would like to see standardized values used for headers and subheaders, these can be defined in the theme and used as default values.

```
theme.chart.header
theme.chart.subheader
```

Something along those lines.  I would like to avoid any component specific text sizing as it isn't maintainable and will make life difficult for accessability and responsive design down the road.

I realize the figma design has some specific values, however those are guidelines and are not meant to be hard requirements.

DEFAULT_CONTAINER_HEIGHT can stay in the component, that doesn't need to be refactored to the theme.

Reply:
Resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1087/commits/94e4d1b1cb9497d56bca5c59a3734595bffb57bd